source/includes/_activities.md

```diff
@@ -72,6 +72,7 @@ expanded | true | Optional expanded json. Returns certificate and any related ex
 include_deactivated_users | true | Optional - default is false. When true activity returned by the end-point will include activity by deactivated users.
 filters[date][from] | "2021-12-31" | Date range FROM date in ISO 8601 format
 filters[date][to] | "2022-02-28" | Date range TO date in ISO 8601 format
+filters[timezone] | "UTC" | Specify Timezone by which to filter activites. See [Rails Timezone names](https://apidock.com/rails/ActiveSupport/TimeZone).
```

Comment:
I would replicate the list here rather

Reply:
and reference this page on the API error response yeah?

Reply:
this is live btw so can you help me fix it